www/grpc/swagger-ui/pactus.swagger.json

```diff
@@ -362,7 +406,7 @@
         ]
       }
     },
-    "/v1/transaction/id/{id}/verbosity/{verbosity}": {
+    "/v1/transaction/get/id/{id}/verbosity/{verbosity}": {
```

Comment:
Why get path?

Reply:
To add verb for this route and be consistent with other routes. Like:
```
      get: "/v1/transaction/get/id/{id}/verbosity/{verbosity}"
      get: "/v1/transaction/calculate_fee/amount/{amount}/payload_type/{payloadType}"
      put: "/v1/transaction/send/data/{data}"

```

Reply:
> To add verb for this route and be consistent with other routes. Like:
> ```
>       get: "/v1/transaction/get/id/{id}/verbosity/{verbosity}"
>       get: "/v1/transaction/calculate_fee/amount/{amount}/payload_type/{payloadType}"
>       put: "/v1/transaction/send/data/{data}"
> 
> ```

I think better change to:

```
/v1/transactions/{id,number,hash and etc}/verbosity/{verbosity}
```